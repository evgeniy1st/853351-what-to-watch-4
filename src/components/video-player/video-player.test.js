import React from "react";
import renderer from "react-test-renderer";
import VideoPlayer from "./video-player.jsx";

it(`render VideoPlayer`, () => {
  const mockFilm =
  {
    image: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    preview: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  };

  const tree = renderer
    .create(
        <VideoPlayer
          src={mockFilm.preview}
          poster={mockFilm.image}
          mouseOver={() => {}}
          mouseLeave={() => { }}
        />)
    .toJSON;

  expect(tree).toMatchSnapshot();
});
