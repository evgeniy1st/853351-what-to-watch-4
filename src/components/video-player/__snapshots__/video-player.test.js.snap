// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render VideoPlayer 1`] = `[Function]`;
